spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/catalog/TestTransformTableChange.java

```diff
@@ -39,6 +39,25 @@ void testTransformRemoveProperty() {
     Assertions.assertEquals("key", gravitinoRemoveProperty.getProperty());
   }
 
+  @Test
+  void testTransformRenameColumn(){
+    String[] oldFiledsName = new String[]{"default_name"};
+    String newFiledName = "new_name";
+
+    TableChange.RenameColumn sparkRenameColumn =
+            (TableChange.RenameColumn) TableChange.renameColumn(oldFiledsName,newFiledName);
+    com.datastrato.gravitino.rel.TableChange gravitinoChange =
+            GravitinoCatalog.transformTableChange(sparkRenameColumn);
+
+    Assertions.assertTrue(
+            gravitinoChange instanceof com.datastrato.gravitino.rel.TableChange.RenameColumn);
+    com.datastrato.gravitino.rel.TableChange.RenameColumn gravitinoRenameColumn =
+            (com.datastrato.gravitino.rel.TableChange.RenameColumn) gravitinoChange;
+
+    Assertions.assertEquals("default_name",gravitinoRenameColumn.getFieldName()[0]);
```

Comment:
should check `oldFiledsName` and gravitinoRenameColumn.getFieldName(),  not just first field